ember-cli-build.js

```diff
@@ -45,8 +45,15 @@ module.exports = function() {
   });
 
   app.import('bower_components/ember/ember-template-compiler.js');
+  app.import('vendor/hint.min.css');
```

Comment:
I will need to compile this. I'll see if I still have the files or make a new one.